### ISSUE TITLE
[BUG] - Code block syntax highlighting has inaccessible color contrast

### ISSUE THREAD

**Issue by gabalafou** [Quansight/Quansight-website#319]
### What site is this for?

Quansight Labs

### Actual behavior

Currently, when running the Deque Chrome Dev Tools Axe audit on a [blog page with code blocks](https://quansight-labs-published-h9vlhh1x8-quansight.vercel.app/blog/portable-python-env), it reports contrast issues in the syntax highlighting.

**Comment by trallard** [comment]
We are currently using [CodeHike for syntax highlighting](https://codehike.org/)

From their configuration docs: https://codehike.org/docs/configuration it uses [`shiki`](https://github.com/shikijs/shiki) so we should be able to change the theme. I do not know which one would meet wcag standards (see list https://codehike.org/docs/configuration#theme-list). Some of which might meet standards:

- GitHub dark 
- Material darker

Else their docs say we can use VSCode themes https://codehike.org/docs/configuration#custom-themes 

But - my guess is many themes will fail on contrast

**Comment by trallard** [comment]
Also - if we're making changes to the config can we add line numbers and a copy-paste button?
